app/scripts/services/import-service.js

```diff
@@ -183,8 +184,36 @@
             if (option === importServiceConflictModal.REPLACE_FILE) {
               var path = ramlRepository.join(directory.path, name);
               var file = ramlRepository.getByPath(path);
+              var promise;
 
-              file.doc.setValue(contents);
+              if (file) {
+                // Load the file if not loaded.
+                promise = file.loaded ? $q.when(file) : ramlRepository.loadFile({path: path});
```

Comment:
I'm guessing that you decided to do so in order to check whether contents has been changed or not, but I think it's redundant and we can skip such check in case of replace.